roles/master-pdf-editor/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+# master PDF Editor installieren
+
+  - name: Installing requirements for Master PDF Editor
+    apt:
+      name: libqt5printsupport5
+
+  - name: Install master pdf editor
+    apt:
+      deb: https://code-industry.net/public/master-pdf-editor-5.8.20-qt5.x86_64.deb
```

Comment:
Die URL oder die Versionsnummer in eine Variable und dann in defaults/main.yml ? Oder ist das overkill?

Reply:
Denke das ist overkill hierfür.